.github/PULL_REQUEST_TEMPLATE/register_did_method.md

```diff
@@ -0,0 +1,47 @@
+## Register a DID Method
+
+### Required Registration Criteria
+
+Please confirm:
+
+- [ ] The DID Method specification defines the identifer syntax.
+- [ ] The DID Method specification defines the supported DID Operations.
+- [ ] The DID Method specification contains a Privacy and Security Considerations.
+
+### Optional Registration Criteria
+
+Consider ensuring:
+
+- [ ] The registration entry contains contact information.
+- [ ] The registration entry contains registry information.
+
+### Registration Process
+
+In order to register a New DID Method, you must add a json file to [./methods](./methods).
+
+Here is an example registration entry:
+
+```jsonc
+{
+  // These fields are required
```

Comment:
```suggestion
  // These fields are required (and this line must be deleted, as JSON doesn't support comments)
```

Reply:
I don't think we need to say this because 1) most people working w/ JSON know this and most registrants have a technical background, and 2) the schema validation will fail and will tell them why if they keep the comments in the file.

Reply:
Is it possible to at least apply the clever CSS that hides the inline comments from a select&copy?  I have problems with including inline comments in examples of any language/syntax that doesn't really support those comments, no matter what ineffective fake-escape is applied (`//`, `--`, `#`, `/*`, `/!` etc.).  If comments are needed in that language (and I do agree that they're helpful here!), then that language needs a revision to support them (which I understand is at least under consideration for the next rev of JSON, but it isn't there yet).

Reply:
> Is it possible to at least apply the clever CSS that hides the inline comments from a select&copy?

No, I don't think it is, because this is Markdown that's rendered in the Github interface (and they don't let you have control over CSS styles IIRC because that can be used as an attack/tracking vector). I'll look into it after merging this PR (I can't test this stuff out w/o it being live on the Github site since this is a PR template, IIRC).

Reply:
Ah, my bad. I didn't look at the filename. :-/  I'll rejoice if GitHub Markdown someday supports such a thing, but I won't hold my breath waiting for it.